src/commands/create-fixture.ts

```diff
@@ -18,7 +18,7 @@ export default async function main(directory: string, overwriteInfo: boolean) {
   let response;
   if (overwriteInfo || !existsSync(jsonFixturePath)) {
     response = await queryPRInfo(prNumber);
-    writeFileSync(jsonFixturePath, scrubDiagnosticDetails(JSON.stringify(response, null, "  ")));
+    writeFileSync(jsonFixturePath, scrubDiagnosticDetails(JSON.stringify(response, null, "  ") + "\n"));
```

Comment:
How about adding:

```ts
const writeJsonToFile = (json: unknown, file: string) =>
    writeFileSync(file, scrubDiagnosticDetails(JSON.stringify(json, null, "  ") + "\n"));
```

and using it everywhere that `writeFileSync` is used?  (Including places that don't use `JSON.stringify`, and places that don't scrub the string, since it's harmless in both cases.)

Reply:
Ah, I see that I missed some newlines: `writeFileSync(..., "{}")` Thanks for catching!

Done :heavy_check_mark: